Python/Product/Debugger/Debugger/PythonThread.cs

```diff
@@ -33,31 +33,23 @@ class PythonThread {
             _name = "";
         }
 
-        public Task StepIntoAsync(CancellationToken ct) {
-            return _process.SendStepIntoAsync(_identity, ct);
-        }
+        public Task StepIntoAsync(CancellationToken ct) =>
+            _process.SendStepIntoAsync(_identity, ct);
```

Comment:
Now I see it, I actually prefer the version with braces.

I'm not so worried about long lines for trivial implementations like these. It's obviously not worth reading, so it doesn't really matter if it goes off the screen IMHO.